HousingFinanceInterimApi/Handler.cs

```diff
@@ -127,6 +132,26 @@ public Handler()
                 reportGateway, transactionGateway, googleFileSettingGateway, googleClientService);
             _moveHousingBenefitFileUseCase = new MoveHousingBenefitFileUseCase(batchLogGateway, batchLogErrorGateway,
                 googleFileSettingGateway, googleClientService);
+
+            var logGroups = new List<string> {
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-load-tenagree",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-cash-file-trans",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-check-cash-files",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-direct-debit",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-direct-debit-trans",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-direct-debit-trans-dem",
+                $"hfs-nightly-jobs-charges-ingest-ecs-task-logs",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-cash-file",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-adjustments-trans",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-action-diary",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-susp-cash",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-susp-hb",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-housing-file",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-housing-file-trans",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-refresh-cur-bal",
+                $"/aws/lambda/housing-finance-interim-api-{environmentName}-refresh-op-bal",
+            };
```

Comment:
Just to confirm: we're not interested in Generate Rent Position?

Reply:
oops. Yes we are. I will add it in